examples/private_set_intersection/client/private_set_intersection.cc

```diff
@@ -80,9 +80,13 @@ int main(int argc, char** argv) {
   ::oak::NodeClient::InitializeAssertionAuthorities();
 
   // Connect to the newly created Oak Node from different clients.
-  auto stub_0 = PrivateSetIntersection::NewStub(::grpc::CreateChannel(
+  auto channel_0 = ::grpc::CreateChannel(
       addr.str(),
-      ::asylo::EnclaveChannelCredentials(::asylo::BidirectionalNullCredentialsOptions())));
+      ::asylo::EnclaveChannelCredentials(::asylo::BidirectionalNullCredentialsOptions()));
+  auto node_client_0 = ::oak::NodeClient(channel_0);
```

Comment:
auto should generally only be used where it improves readability, so I would prefer this here:

::oak::NodeClient node_client_0(channel_0);

We also don't necessarily need to use the ::oak:: prefix within our own namespace, it just adds clutter.

Reply:
Done, left ::oak prefix around for now, but happy to do a cleanup at some point.